src/main/java/org/sonar/plugins/stash/issue/SonarQubeIssue.java

```diff
@@ -1,45 +1,30 @@
 package org.sonar.plugins.stash.issue;
 
-public class SonarQubeIssue {
+public class SonarQubeIssue extends Issue {
 
-  private final String key;
-  private final String message;
-  private final String severity;
   private final String rule;
-  private final long line;
-  private final String path;
-
+  private final String message;
+  
   public SonarQubeIssue(String key, String severity, String message, String rule, String path, long line) {
-    this.key = key;
-    this.message = message;
-    this.severity = severity;
+    super(key, severity, path, line);
     this.rule = rule;
-    this.path = path;
-    this.line = line;
-  }
-
-  public String getKey() {
-    return key;
-  }
-
-  public String getSeverity() {
-    return severity;
-  }
-
-  public String getMessage() {
-    return message;
+    this.message = message;
   }
 
   public String getRule() {
     return rule;
   }
-
-  public long getLine() {
-    return line;
+  
+  public String getMessage() {
+    return message;
   }
+    
+  public String printIssueMarkdown(String sonarQubeURL) {

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1161)